claims/verifier.go

```diff
@@ -0,0 +1,83 @@
+package claims
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// Verifier bundles a number Assertions
+type Verifier struct {
+	Assertions Assertions
+}
+
+// NewVerifier initializes a new verifier
+func NewVerifier() *Verifier {
+
+	return &Verifier{
+		Assertions: make(map[Claim][]Assertion),
+	}
+
+}
+
+// Add adds an Assertion for a given Claim
+func (v *Verifier) Add(c Claim, a Assertion) *Verifier {
```

Comment:
Currently for supporting the [builder pattern](https://en.wikipedia.org/wiki/Builder_pattern), but there is no reason that could not be immutable. I could change it, I suppose.